lamindb_setup/core/upath.py

```diff
@@ -484,3 +485,65 @@ def create_path(path: UPath, access_token: Optional[str] = None) -> UPath:
         secret=credentials["secret"],
         token=credentials["token"],
     )
+
+
+def get_stat_file_cloud(stat: Dict) -> Tuple[int, str, str]:
```

Comment:
We need tests for this, @bpenteado!

In lamindb, this was covered, now, I fear, it's uncovered by CI.

My guess is that there were no fine-grained unit tests in lamindb for this, but coverage came through more high-level functionality.

Given this is in lamindb-setup now and the interfaces are used by laminhub and lamindb, we need a more fine-grained test that reflects needs in both downstream repos (they should be the same if we design the code well).

Reply:
Here is an issue for this: https://github.com/laminlabs/lamindb-setup/issues/665